modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -31,8 +31,13 @@ def perform(claim_id, middle_initial, perform_async = true) # rubocop:disable Me
           log_job_progress(claim_id,
                            '526EZ PDF generator PDF string returned')
 
```

Comment:
Should we add logging to indicate the 526 PDF generator is being mocked when the setting is set to true, similar to logging in `mock_aws_service`?